docs/iris/src/further_topics/metadata.rst

```diff
@@ -0,0 +1,987 @@
+.. _metadata:
+
+********
+Metadata
+********
+
+This section provides a detailed overview of how your metadata is managed
+within Iris. In particular, it discusses what metadata is, where it fits
+into Iris, and more importantly how you can create, access, manipulate,
+and analyse your metadata.
+
+All the finer details covered here may not be entirely relevant to your use
+case, but it's here if you ever need it. In fact, you may want to skip
+straight ahead to :ref:`richer metadata`, and take it from there.
+
+
+Introduction
+============
+
+So far we've introduced several classes in Iris that care about your ``data``,
+and also your ``metadata`` i.e., `data about data`_. Primarily, these classes
+are the :class:`~iris.cube.Cube`, the :class:`~iris.coords.AuxCoord`, and the
+:class:`~iris.coords.DimCoord`, all of which should be familiar to you now.
+
+As discussed in :ref:`iris_data_structures`, Iris draws heavily from the
+`NetCDF CF Metadata Conventions`_ as a source for its data model, thus building
+on the widely recognised and understood terminology defined within those
+`CF Conventions`_ by the scientific community.
+
+Iris models several other classes of `CF Conventions`_ metadata apart from
+those mentioned above. Such as the :class:`~iris.coords.AncillaryVariable`
+(see `Ancillary Data`_ and `Flags`_), the :class:`~iris.coords.CellMeasure`
+(see `Cell Measures`_), and also the :class:`~iris.aux_factory.AuxCoordFactory`
+(see `Parametric Vertical Coordinate`_).
+
+Collectively, the asforementioned classes will be known here as the Iris
+`CF Conventions` classes.
+
+.. hint::
+
+    If there are any `CF Conventions`_ metadata missing from Iris that you
+    care about, then please let us know by raising a `GitHub Issue`_ on
+    `SciTools/iris`_
+
+
+Common Metadata
```

Comment:
As I mentioned in my main comment:

All the info is here, but I found it easier to understand the table once I'd understood what is meant by metadata.  So I'd introduce what a metadata is first, before one tries to read it.  So, I’d move the table to the end of the section, and the example describing it immediately before.
Also:
* its ambiguous as to whether "var_name" and "circular" are the only Iris specific terms.
* In the table we have "standard name" is this the same as the "standard_name" in the pros ? 
* Why do some of the metadata members listed in the table have underscores in them, but others don't.  Are they the same things?  Is one a method?
* And - what does the red represent - I've been wondering this throughout our documentation.


```suggestion
Common Metadata
===============

Each of the Iris `CF Conventions`_ classes use  **metadata** to define them and give them meaning.

A small collective of individual **metadata members** can be used to define the
Iris `CF Conventions`_ classes.  All the members reference specific `CF Conventions`_ terms, 
except for  :attr:`~iris.coords.DimCoord.var_name` and :attr:`~iris.coords.DimCoord.circular` which are Iris specific terms.  The individual metadata members used to define the various Iris `CF Conventions`_ classes are shown in :numref:`metadata members table` . 

As :numref:`metadata members table` highlights, **specific** metadata is used to
define and represent each of the Iris `CF Conventions`_ classes. Thus the
metadata can be used to easily **identify**, **compare** and **differentiate**
between individual class instances.

For example, the collective metadata used to define an
:class:`~iris.coords.AncillaryVariable` are the ``standard_name``, ``long_name``,
``var_name``, ``units``, and ``attributes`` members. Note that, these are the
actual `data attribute`_ names of the metadata members on the Iris class.

```
.. and now have Table 1.  Should the member names all have "_"'s?